CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
 Add:  POST /v2/entities - entity creation under version 2 (Issue #970)
 Add:  API version 2 only supports JSON so we can now distinguish between STRINGS, NUMBERS (all treated as floats for now),
       and BOOLEANS (true, false), for the VALUE for ContextAttribute and Metadata (No Issue)
+Fix:  Fixed a bug about error handling for API version 2 (Issue #1087)
```

Comment:
The bug was already in 0.22.0? I though that it was related with #970 (part of 0.23.0), thus it shouldn't be included in the changelog (as there is no change regarding with 0.22.0). But maybe I'm wrong...